sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/objects.scala

```diff
@@ -121,9 +121,7 @@ object EliminateMapObjects extends Rule[LogicalPlan] {
 object ObjectSerializerPruning extends Rule[LogicalPlan] {
 
   /**
-   * Collects all struct types from given data type object, recursively. Supports struct and array
-   * types for now.
-   * TODO(SPARK-26847): support map type.
+   * Collects all struct types from given data type object, recursively.
    */
   def collectStructType(dt: DataType, structs: ArrayBuffer[StructType]): ArrayBuffer[StructType] = {
```

Comment:
Visible for testing?

Reply:
Not sure if it is definitely needed. Not see such annotation in scala codes, if I'm not missing anyone.

Reply:
+1 for @maropu 's suggestion. It's helpful. For Scala, Apache Spark commonly uses `Visible for testing` in the function description comment. You can search `Visible for testing` from the code base.

Reply:
I see. I thought @maropu was suggesting `@VisibleForTesting`. Will add a comment for it.